apis/concierge/authentication/v1alpha1/types_jwt.go.tmpl

```diff
@@ -57,8 +57,9 @@ type JWTTokenClaims struct {
 // signature, existence of claims, etc.) and extract the username and groups from the token.
 //
 // +genclient
+// +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-// +kubebuilder:resource:categories=pinniped;pinniped-authenticator;pinniped-authenticators
+// +kubebuilder:resource:categories=pinniped;pinniped-authenticator;pinniped-authenticators,scope=Cluster
 // +kubebuilder:printcolumn:name="Issuer",type=string,JSONPath=`.spec.issuer`
 type JWTAuthenticator struct {
```

Comment:
I don't suppose these types should now be called `ClusterXXX` (e.g., `ClusterJWTAuthenticator`) now that they are cluster-scoped?

Reply:
I don't think so because there is no namespaced version now.  i.e. `Node` not `ClusterNode`.